packages/headless/doc-parser/use-cases/product-recommendation.ts

```diff
@@ -0,0 +1,39 @@
+import {ActionLoaderConfiguration} from '../src/headless-export-resolvers/action-loader-resolver';
+import {ControllerConfiguration} from '../src/headless-export-resolvers/controller-resolver';
+import {EngineConfiguration} from '../src/headless-export-resolvers/engine-resolver';
+
+const controllers: ControllerConfiguration[] = [
+  {
+    initializer: 'buildContext',
+    samplePaths: {
+      react_fn: [
+        'packages/samples/headless-react/src/components/context/context.ts',
+      ],
+    },
+  },
```

Comment:
I left out most of the controllers to start off because their interfaces need to be converted to the form that api-extactor needs to be able to resolve them.